jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -1050,11 +1074,12 @@ declare namespace DataTables {
         isDataTable(table: string): boolean;
 
         /**
-        * Get all DataTables on the page
+        * Get all DataTable tables that have been initialised - optionally you can select to get only currently visible tables and / or retrieve the tables as API instances.
         *
-        * @param visible Get only visible tables
+        * @param visible As a boolean value this options is used to indicate if you want all tables on the page should be returned (false), or visible tables only (true).
+        * Since 1.10.8 this option can also be given as an object.
         */
-        tables(visible?: boolean): DataTables.DataTable[];
+        tables(visible?: boolean | ObjectTablesStatic): DataTables.DataTable[] | DataTables.DataTable;
```

Comment:
though i am not sure, but seems like the documentation suggests that passing an object will result in a DataTable, so this should be:

```ts
tables(visible?: boolean): DataTables.DataTable[];
tables(visible?: ObjectTablesStatic): DataTables.DataTable;
```